content/docs/searching.md

```diff
@@ -301,14 +301,32 @@ To some users these older results accumulating are an annoyance -- their data ma
 The following commands, available through the Search menu or keyboard shortcuts, perform a search without invoking a dialog, because they search for the previous search target or for a word or selection in the current document:
 
 * **Find Next / Find Previous** Repeat searching the current search target, either down or up.
-
-* **Next Search Result / Previous Search Result** Jump to the next or previous search result recorded in the Search Results Window. The Search Results Window is created in response to any of the dialog-based **Find All** commands. If it exists, you can use  **Search  -&gt;  Search Results Window** to make it visible and to switch the input focus between the Search Results Window and the current document.
-
-* **Find (volatile) Next / Find (volatile) Previous** Attempt to find the word the caret is in, or the selected text, down or up. The searched word or selection is not remembered in the find history, and the search will not be repeatable with **Find Next / Find Previous**. That's why it's called volatile.
-
+    - The "current search target" is whatever _Find what_ string was most-recently active from either the Find/Replace dialog or from the **Select and Find Next / Select and Find Previous**
+* **Next Search Result / Previous Search Result** Jump to the next or previous search result recorded in the Search Results Window.
+    - The Search Results Window is created in response to any of the dialog-based **Find All** commands.
+    - If such a search has been run in this instance of Notepad++, but the Search Results Window it is not visible, you can use  **Search > Search Results Window** to make it visible; if it is already visible, that command will switch the input focus between the Search Results Window and the current document.
 * **Select and Find Next / Select and Find Previous** Attempt to find the word the caret is in, or the selected text, down or up. The searched word or selection is remembered in the find history, and the search can be repeated with **Find Next / Find Previous**.
-
-All dialog-free search actions respect the current search options set in the Find dialog like Match case or Wrap around.
+    - The sequence of events is described below:
+        - all mentions of the _Find_ window in this sequence are still true even if the _Find_ window is not currently visible
+        - copies the selected text to _Find what_ box of _Find_ window, then uses it
+            - if there is no selection and the caret is just at a single character position, it uses the word that the caret position is a part of (if any)
+            - if the caret is in whitespace with no selection active, it will not perform a search, and the caret will remain where it was
+        - uses _Match case_ setting from _Find_ window
+        - uses _Match whole word only_ setting from _Find_ window
+        - uses _Wrap around_ setting from _Find_ window
+        - uses _Search mode_ = _Normal_
```

Comment:
Suggestion:  `-considers Search mode to be Normal` (regardless of its current setting in the Find window)

Reply:
done